resources/home/dnanexus/generate_workbook/tests/test_excel.py

```
@@ -0,0 +1,161 @@
+import os
+import sys
+import pytest
+from pathlib import Path
+import openpyxl
+
+sys.path.append(os.path.abspath(
+    os.path.join(os.path.realpath(__file__), '../../')
+))
+
+from utils.excel import excel
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)